[2025-01-10T17:21:04.302+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T17:21:04.314+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:20:00+00:00 [queued]>
[2025-01-10T17:21:04.322+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:20:00+00:00 [queued]>
[2025-01-10T17:21:04.323+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T17:21:04.336+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 17:20:00+00:00
[2025-01-10T17:21:04.341+0000] {standard_task_runner.py:72} INFO - Started process 1427 to run task
[2025-01-10T17:21:04.344+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T17:20:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmp6vd3riq5']
[2025-01-10T17:21:04.347+0000] {standard_task_runner.py:105} INFO - Job 155: Subtask append_to_database
[2025-01-10T17:21:04.385+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:20:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T17:21:04.460+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T17:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T17:20:00+00:00'
[2025-01-10T17:21:04.461+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T17:21:20.233+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T17:21:20.235+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T17:21:20.245+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T17:21:20.247+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T17:20:00+00:00, execution_date=20250110T172000, start_date=20250110T172104, end_date=20250110T172120
[2025-01-10T17:21:20.278+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T17:21:20.298+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T17:21:20.301+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
